packages/core/android/build.gradle

```diff
@@ -16,11 +16,16 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdk safeExtGet('compileSdkVersion', 33)
+    compileSdk safeExtGet('compileSdk', 34)
+
+    def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION.tokenize('.')[0].toInteger()
+    if (agpVersion >= 8) {
+        namespace = "com.adobe.marketing.mobile.reactnative"
```

Comment:
Let use
namespace = "com.adobe.marketing.mobile.reactnative.core"
?

Reply:
The `namespace` should be consistent with the project's base package name. If we want to use a different namespace, we will need to refactor the entire project. Also, the core wrapper actually includes core, lifecycle, identity, and signal extensions, so I don't think `com.adobe.marketing.mobile.reactnative.core` is a better option.